addon/components/dialog.hbs

```diff
@@ -0,0 +1,43 @@
+{{#if @isOpen}}
+  {{#let (element this.tagName) as |Tag|}}
+    {{#in-element this.$portalRoot insertBefore=null}}
+      <Tag
+        id={{this.guid}}
+
+        role="dialog"
+        aria-modal="true"
+
+        {{focus-trap}}
```

Comment:
If you _wanted_ to support the `initialFocus` argument that's mentioned in the PR description, it could be passed down into the underlying `focus-trap` library's options https://github.com/josemarluedke/ember-focus-trap#with-focus-trap-options

Reply:
Good shout. I do plan to implement that. Just wasn’t a priority in this PR. Will keep that in mind. Thanks.